### PR TITLE
add 2024 kcseu scaffold page

### PR DESCRIPTION
**Pull request by moficodes**
WIP:
fixes: #460

### CODE REVIEW

File: content/en/events/2024/kcseu/faq.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+- [I have an idea for a session](#i-have-an-idea-for-a-session)
+- [My SIG wants to meet at the Summit](#my-sig-wants-to-meet-at-the-summit)
+- [Will there be a doc sprint?](#will-there-be-a-doc-sprint)
+- [Will there be a New Contributor Workshop?](#will-there-be-a-new-contributor-workshop)
+- [Can I bring a guest to the Social?](#can-i-bring-a-guest-to-the-social)
+- [I have a question! How do I contact the event staff?](#i-have-a-question-how-do-i-contact-the-event-staff)
+
+### When is the summit taking place?
+
+TBD
```
Comment (mfahlandt) [inline content/en/events/2024/kcseu/faq.md:22]: 19.04.2024

File: content/en/events/2024/kcseu/schedule.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+## Doc Sprint
+
+SIG-Docs will have a full-day Documentation working session during the Summit.
+If you contribute to the docs -- and who doesn't -- or you want to learn to
+do it better, join them.
+
+## CFP
+
+If you would like to submit a formal session, please use the [CFP Form]().
```
Comment (mfahlandt) [inline content/en/events/2024/kcseu/schedule.md:27]: Remove Form for now and Replace with:

The CFP will open soon and will cover presentation, workshop, and collaboration style sessions.  
Unconference sessions should use the GitHub issue referenced above.

**Comment by natalisucks** [comment]
@moficodes thanks for starting this, Mofi! our standard across Kubernetes docs is that page weights go up via an increment of 10 (see one issue this references regarding the need to update autogenerated code [#37486](https://github.com/kubernetes/website/issues/37486)). while we haven'g got this consistent throughout our docs yet, i'd say we ought to follow that standard for the contributor site

File: content/en/events/2024/kcseu/faq.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+
+March 19, 2024
+
+### Where is the summit located?
+
+1 Place de la Porte de Versailles, 75015 Paris, France
+
+### Will I be able to attend remotely?
+
+No, based on our experience with Kubecon NA 2021, we do not expect to be able
```
Comment (jberkus) [inline content/en/events/2024/kcseu/faq.md:30]:
```suggestion
No, based on our experience with Kubecon EU 2023, we do not expect to be able
```

File: content/en/events/2024/kcseu/faq.md [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+Most of the Summit will be unconference-style, so you will have an opportunity
+to propose a session closer to the Summit itself. If you have an idea for a talk,
+workshop or panel please post your idea to the unconference issue or the CFP
+form once they become available.  
+
+### My SIG wants to meet at the Summit
+
+We do not know yet if we will be able to request additional space outside of
+the colo event space on Tuesday, April 18th. We will update this page and send
+out community wide messages about it once the details are available.
```
Comment (jberkus) [inline content/en/events/2024/kcseu/faq.md:83]:
```suggestion
We plan to include SIG meetings as part of the schedule for the Summit.  Please watch for the CfP, and submit your proposed SIG meeting as a Summit session.
```

File: content/en/events/2024/kcseu/schedule.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+## Steering AMA
+
+The Summit will start off with an AMA with members of the Kubernetes Steering
+Committee.  Bring any and all questions about how we're running the project,
+ongoing efforts & issues, and whatever you'd like to discuss with Steering.
+
+## Unconference
+
+TBD
```
Comment (jberkus) [inline content/en/events/2024/kcseu/schedule.md:17]:
```suggestion
The Summit will include Unconference sessions, probably in the afternoon.  If important discussions arise after the close of the CfP, plan to pitch them as Unconference sessions.  More details to come.
```

File: content/en/events/2024/kcseu/schedule.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+
+## CFP
+
+The CFP will open soon and will cover presentation, workshop and collaboration style sessions. Unconference sessions should use the Github issue referenced above.
+
+## Schedule
+
+{{% schedule
+  text="Kubernetes Contributor Summit Europe 2024"
+%}}
```
Comment (jberkus) [inline content/en/events/2024/kcseu/schedule.md:33]: Should probably remove this block until it's live.

File: content/en/events/2024/kcseu/social.md [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+## Bringing A Guest
+
+Contributors will be able to bring a single family member or partner to the
+Social. This does not include coworkers or casual friends; our intention is
+to support domestic partners and close family of contributors by including
+them.
+
+**NOTE:** Guests must adhere to all KubeCon health & safety requirements.
+
+To bring a guest, please email community@kubernetes.io with their information.
```
Comment (jberkus) [inline content/en/events/2024/kcseu/social.md:22]:
```suggestion
More information on Guest registration to come, closer to the event.
```

File: content/en/events/2024/kcseu/social.md [hunk 8]
```diff
@@ -0,0 +1,22 @@
+---
+title: Contributor Social
+type: docs
+weight: 5
+---
+
+TBD
```
Comment (jberkus) [inline content/en/events/2024/kcseu/social.md:7]:
```suggestion
The Summit will include a social event open only to Contributors (and their family members, see Guests below).  This event will take place somewhere near the Summit venue, and will include food, drink, entertainment, and games.  More information will be available once the venue is booked.

**Contributors must be registered for the Summit in order to attend the Social.**
```

**Review by jberkus — CHANGES_REQUESTED** [review]

File: content/en/events/2024/kcseu/faq.md [hunk 9]
```diff
@@ -0,0 +1,113 @@
...
+
+Kubernetes Orgs:
+
+<ul>
+<li><a href="https://github.com/kubernetes" rel="noopener noreferrer" target="_blank">kubernetes</a></li>
+<li><a href="https://github.com/kubernetes-client" rel="noopener noreferrer" target="_blank">kubernetes-client</a></li>
+<li><a href="https://github.com/kubernetes-csi rel="noopener noreferrer" target="_blank">kubernetes-csi</a></li>
+<li><a href="https://github.com/kubernetes-sigs" rel="noopener noreferrer" target="_blank">kubernetes-sigs</a></li>
+</ul>
+
```
Comment (mrbobbytables) [inline content/en/events/2024/kcseu/faq.md:61]:
```suggestion
<li><a href="https://github.com/kubernetes-sigs" rel="noopener noreferrer" target="_blank">kubernetes-sigs</a></li>
<li><a href=“https://github.com/etcd-io” rel=“noopener noreferrer” target=“_blank”>etcd-io</a></li>
</ul>

```

etcd-io is now a k8s org

File: content/en/events/2024/kcseu/faq.md [hunk 10]
```diff
@@ -0,0 +1,113 @@
...
+or partner to the social (a "plus one"). 
+[See the Social page to request a guest pass](https://www.kubernetes.dev/events/2023/kcseu/social/).
+
+### I have a question! How do I contact the event staff?
+
+You can [email us] or pop into the
+<a href="https://kubernetes.slack.com/messages/contributor-summit" rel="noopener noreferrer" target="_blank">
+#contributor-summit</a> Slack channel to ask a question. :)
+
+[email us]: mailto:community@kubernetes.io
```
Comment (mrbobbytables) [inline content/en/events/2024/kcseu/faq.md:113]:
```suggestion
[email us]: mailto:summit-team@kubernetes.io
```

**Review by mrbobbytables — CHANGES_REQUESTED** [review]
Thanks for stubbing it out, just have a few small changes to update for 2024 👍

**Comment by jberkus** [comment]
@natalisucks we haven't done that with the Contributor Site, though.  We probably should, BUT that should be a separate PR from this one.  We could also think about trying to set things up so that we don't have to retro edit all the pages.  But again, I don't want to hold this PR on that.

File: content/en/events/2024/kcseu/schedule.md (hunk 6 above)
Comment (jberkus) [reply content/en/events/2024/kcseu/schedule.md:33]: @moficodes if you can fix this, then I can merge this PR.

**Comment by moficodes** [comment]
@jberkus @mrbobbytables I think thats all the comments resolved.

**Comment by natalisucks** [comment]
/lgtm
pushing this live ASAP is worthwhile then we can update other text/grammar fixes as needed

**Pull request merged**